internal/node_loader.js

```diff
@@ -67,6 +67,11 @@ function resolveToModuleRoot(path) {
  * See https://github.com/bazelbuild/bazel/issues/3726
  */
 function loadRunfilesManifest(manifestPath) {
+  // If the runfiles directory exists, we're not running on Windows, so
+  // we don't need the runfiles manifest
+  if (fs.existsSync(process.env.RUNFILES)) {
+    return;
+  }
   if (!fs.existsSync(manifestPath)) {
```

Comment:
either remove this second conditional, or make it throw an error if the manifestPath doesn't exist.